src/components/font_editor/font_editor.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  Component,
+  onWillStart,
+  onWillUpdateProps,
+  useExternalListener,
+  useRef,
+  useState,
+} from "@odoo/owl";
+import { DEFAULT_FONT_SIZE, FONT_SIZES } from "../../constants";
+import { clip } from "../../helpers/index";
+import { setStyle } from "../../registries/index";
+import { SpreadsheetChildEnv, Style } from "../../types/index";
+import { css } from "../helpers/css";
+import { MenuState } from "../menu/menu";
+
+interface State {
+  menuState: MenuState;
+}
+
+interface Props {
+  callback: () => void;
+}
+
+// -----------------------------------------------------------------------------
+// TopBar
+// -----------------------------------------------------------------------------
+css/* scss */ `
+  .o-spreadsheet-topbar {
+    /* Toolbar + Cell Content */
+    .o-topbar-toolbar {
+      input.o-font-size {
+        height: 20px;
+        width: 23px;
+        border: none;
+        background-color: rgba(0, 0, 0, 0);
+      }
+      input[type="number"] {
+        -moz-appearance: textfield;
+      }
+      input::-webkit-outer-spin-button,
+      input::-webkit-inner-spin-button {
+        -webkit-appearance: none;
+      }
```

Comment:
It was my suggestion - the idea is to hide the arrows that you can usually see on `input[type="number"]`  because it felt a bit too cramped  to see both those arrows AND the arrow for the dropdown.